app/views/hiring_staff/schools/show.html.haml

```diff
@@ -5,7 +5,7 @@
     %h1.heading-large
       = t('schools.jobs.index', school: @school.name)
     - if @multiple_schools
-      = link_to t('sign_in.organisation.change'), new_azure_path
+      = link_to t('sign_in.organisation.change'), auth_dfe_callback_path
```

Comment:
Will the Azure users who have multiple schools lose the ability to choose organisations with our 'choose organisation' page? 

I'm not overly concerned if they do because none of them have logged in in at least 8 weeks and we're about to migrate them away. They can still use the 'Choose organisation' feature when signing in but will lose this ability to change organisation once in. They can always sign out and sign back in if we get any support requests in the few days inbetween now and the migration where they can't use it at all.

Reply:
Yeah you're right there could be some time where Azure users can't switch organisation. When do you think we're gonna migrate over? Maybe we could delay merging this change until then?

Reply:
In the next 2 days I hope!